doc/api.md

```diff
@@ -176,32 +176,46 @@ applies to autoprovisioned attributes and is also available at JEXL context with
+
+It is also possible to configure how each of the measures obtained from the device is mapped to different attributes.
+The name and type of the attribute is configured by the user (globally for all the devices in the group or in a per
+device preprovisioning). Device measures can have four different behaviors:
 
 -   **`attributes`**: Are measures that are pushed from the device to the IoT agent. This measure changes will be sent
     to the Context Broker as updateContext requests over the device entity. NGSI queries to the context broker will be
     resolved in the Broker database. For each attribute, its `name` and `type` must be provided. Additional `metadata`
-    is optional.
+    is optional. They are called internally as _active attributes_.
 
 -   **`lazy`**: Passive measures that are pulled from the device to the IoT agent. When a request for data from a lazy
-    attribute arrives to the Context Broker, it forwards the request to the Context Provider of that entity, in this
-    case the IoT Agent. The IoT Agent will then ask the device for the information needed, transform that information to
-    a NGSI format and return it to the Context Broker. This operation will be synchronous from the customer perspective:
-    the Context Broker won't return a response until the device has returned its response to the IoT Agent. For each
-    attribute, its `name` and `type` must be provided.
+    attribute arrives to the Context Broker, it forwards the request to the IoT Agent (that behaves as NGSI Context
+    Provider for all the lazy attributes or commands). The IoT Agent will then ask the device for the information
+    needed, transform that information to a NGSI format and return it to the Context Broker. This operation will be
+    synchronous from the customer perspective: the Context Broker won't return a response until the device has returned
+    its response to the IoT Agent. For each attribute, its `name` and `type` must be provided. They are called
+    internally as _lazy attributes_.
 
 -   **`static`**: It is static attributes that are persisted in the Context Broker. They are not updated by the device,
     but they can be modified by the user. They are useful to store information about the device that is not updated by
     the device itself. For instance, a `location` static attribute is can be used to store the location of a fixed
     device.
 
--   **`commands`**: Commands are actions that can be invoked in the device. They are similar to attributes, but they are
-    not updated by the device. They are updated by the Context Broker, and the IoT Agent will be in charge of
-    translating the updateContext request to the proper action in the device. Two additional attributes are created for
-    each command: `status` and `info`. For each command, its `name` and `type` must be provided.
+-   **`commands`**: Commands are actions that can be invoked in the device. They are entity attributes, but they are not
```

Comment:
Maybe a link to the recently added section about Command Execution would be useful at this point.

Reply:
Added here: https://github.com/telefonicaid/iotagent-node-lib/pull/1470/commits/fe57c79fedcca7b44ffb7fa7b2e26a7df0fcd402